lib/transforms/base-scss.js

```diff
@@ -24,7 +24,10 @@ module.exports = function (options, output) {
 		]);
 		output.sassLoader = {
 			sourcemap: true,
-			includePaths: [ path.resolve('./bower_components') ],
+			includePaths: [
+				path.resolve('./bower_components'),
+				path.resolve('./node_modules')
```

Comment:
I will try with the limitation to `/@financial-times` did run into some issues when previously attempting that, but I think they were caused by other things.

I'm presuming that webpack will only try and find things that are referenced in the client side code, so it shouldn't be looking at that type of third party code, but never underestimate what people may try and do if they can....